### PR TITLE
DocTool rule R - 02366 County drop downs implementation.

### PR DESCRIPTION
**Pull request by LotharII**
[#483]

Created ValidCounty validator to prevent appearing 'State of California' government entity type in the DB.

## Description
All county drop downs are populated from the code table GOVERNMENT ENTITY TYPE, excluding the value 'State of California'.

## Jira Issue link
https://osi-cwds.atlassian.net/browse/INT-483

## Tests
- [x] I have included unit tests 
- [ ] I have included functional tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the google code style of this project.
- [x] I have ran all tests for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check SonarQube after the build completes, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: src/main/java/gov/ca/cwds/data/persistence/cms/DrmsDocumentTemplate.java [hunk 1]
```diff
@@ -57,6 +58,7 @@
   @JsonDeserialize(using = CmsSystemCodeDeserializer.class)
   @Type(type = "short")
   @Column(name = "GVR_ENTC")
+  @ValidCounty
```
Comment (tparkerosi) [inline src/main/java/gov/ca/cwds/data/persistence/cms/DrmsDocumentTemplate.java]: Is this really part of county or government entity?

File: src/test/java/gov/ca/cwds/data/persistence/cms/DrmsDocumentTemplateTest.java [hunk 2]
```diff
@@ -32,6 +36,20 @@ public void testEmptyConstructor() throws Exception {
...
   }
 
+
+  /**
+   *  R - 02366 County drop douwns
+   */
+  @Test
+  public void testCmsCaseCounty(){
+    ValidatorFactory factory = Validation.buildDefaultValidatorFactory();
+    validator = factory.getValidator();
```
Comment (tparkerosi) [inline src/test/java/gov/ca/cwds/data/persistence/cms/DrmsDocumentTemplateTest.java]: What's vp? It doesn't make sense to me. We should avoid abbreviations and use intention revealing names.

Comment (LotharII) [reply src/test/java/gov/ca/cwds/data/persistence/cms/DrmsDocumentTemplateTest.java]: Shure, it was a poor copy paste. Refactored.

File: src/main/java/gov/ca/cwds/data/persistence/cms/DrmsDocumentTemplate.java (hunk 1 above)
Comment (LotharII) [reply src/main/java/gov/ca/cwds/data/persistence/cms/DrmsDocumentTemplate.java]: Removed validation from the non address related fields.

**Pull request merged**